docs/dns-policy.md

```diff
@@ -84,6 +84,18 @@ spec:
     name: prod-web
     group: gateway.networking.k8s.io
     kind: Gateway
+  healthCheck:
+    allowInsecureCertificates: true
```

Comment:
this still looks wrong? in the CRD is `allowInsecureCertificate` https://github.com/Kuadrant/multicluster-gateway-controller/pull/409/files#diff-72079cb887312d7bc89d463be464b6882d63251f812ef1efbef663182e4a6d4dR34 but here it is `allowInsecureCertificates`